### DISCUSSION TITLE
Update playlist via API ?

### DISCUSSION THREAD

**Discussion by gaetanars** [pixelit-project/PixelIt#219] [Q&A] [answered]
Dear all,

I try to use my Pixel IT without NodeRed and I don't see in the api documentation what endpoint to use for update screens playlist ?
Is it a special endpoint ?

Thx

**Comment by foorschtbar** [comment, marked as answer]
The Playlist is a Feature from the Node-RED PixelIt Core Node. Without it, you can "only" shedule your different screens with your own logic. The PixelIt matrix accepts only one screen at the time.

**Reply by gaetanars** [reply]
Thx for your reply. I understand that is simpler to stay with my NodeRed instance
